rest_framework/schemas/openapi.py

```diff
@@ -442,18 +442,11 @@ def _map_field(self, field):
                 content['format'] = field.protocol
             return content
 
-        # DecimalField has multipleOf based on decimal_places
         if isinstance(field, serializers.DecimalField):
-            content = {
-                'type': 'number'
+            return {
+                'type': 'string',
+                'format': 'decimal'
```

Comment:
The OpenAPI spec allows for specifying a format despite it not being "official". See https://github.com/OAI/OpenAPI-Specification/issues/845#issuecomment-378139730 for additional discussion.